packages/core/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@builder.io/sdk",
-  "version": "1.1.27",
+  "version": "1.1.28-5",
```

Comment:
should this be pointing to a dev release instead of a 1.1.29 ?

Reply:
Correct, was waiting for review before cutting a stable release, I'll do this next, thanks Shyam